Makefile

```diff
@@ -66,22 +80,31 @@ prod: ## Start Production server
 start: install build ## Start server
 	@yarn run start
 
-# -- Docker --
+# -- Container Runtime --
 
 .PHONY: cbuild
-cbuild: ## Build docker image
-	docker build -t ${REGISTRYNS}/${BINNAME}-builder:${VERSION} --cache-from ${REGISTRYNS}/${BINNAME}-builder:latest --target build_base                             --build-arg VERSION=${VERSION} .
-	docker tag ${REGISTRYNS}/${BINNAME}-builder:${VERSION} ${REGISTRYNS}/${BINNAME}-builder:latest
+cbuild: ## Build container image	
+	@echo "Building image with $(CONTAINER_TOOL)"
+
+	${CONTAINER_TOOL} build -t ${REGISTRYNS}/${BINNAME}-builder:${VERSION} --cache-from ${REGISTRYNS}/${BINNAME}-builder:latest --target build_base                             --build-arg VERSION=${VERSION} .
+	${CONTAINER_TOOL} tag ${REGISTRYNS}/${BINNAME}-builder:${VERSION} ${REGISTRYNS}/${BINNAME}-builder:latest
  
-	docker build -t ${REGISTRYNS}/${BINNAME}:${VERSION}         --cache-from ${REGISTRYNS}/${BINNAME}-builder:latest --cache-from ${REGISTRYNS}/${BINNAME}:latest    --build-arg VERSION=${VERSION} --build-arg "MOVE2KUBE_UI_GIT_COMMIT_HASH=${GIT_COMMIT}" --build-arg "MOVE2KUBE_UI_GIT_TREE_STATUS=${GIT_DIRTY}" .
-	docker tag ${REGISTRYNS}/${BINNAME}:${VERSION} ${REGISTRYNS}/${BINNAME}:latest
+	${CONTAINER_TOOL} build -t ${REGISTRYNS}/${BINNAME}:${VERSION}         --cache-from ${REGISTRYNS}/${BINNAME}-builder:latest --cache-from ${REGISTRYNS}/${BINNAME}:latest    --build-arg VERSION=${VERSION} --build-arg "MOVE2KUBE_UI_GIT_COMMIT_HASH=${GIT_COMMIT}" --build-arg "MOVE2KUBE_UI_GIT_TREE_STATUS=${GIT_DIRTY}" .
+	${CONTAINER_TOOL} tag ${REGISTRYNS}/${BINNAME}:${VERSION} ${REGISTRYNS}/${BINNAME}:latest
 
 .PHONY: cpush
-cpush: ## Push docker image
+cpush: ## Push container image
+	@echo "Pushing image with $(CONTAINER_TOOL)"
 	# To help with reusing layers and hence speeding up build
-	docker push ${REGISTRYNS}/${BINNAME}-builder:${VERSION}
-	docker push ${REGISTRYNS}/${BINNAME}:${VERSION}
+	${CONTAINER_TOOL} push ${REGISTRYNS}/${BINNAME}-builder:${VERSION}
+	${CONTAINER_TOOL} push ${REGISTRYNS}/${BINNAME}:${VERSION}
 
 .PHONY: crun
-crun: ## Run using docker
-	docker run --rm -it -p 8080:8080 -v ${PWD}/data:/move2kube-api/data -v /var/run/docker.sock:/var/run/docker.sock quay.io/konveyor/move2kube-ui:latest
+crun: ## Run using container image
+	@echo "Running image with $(CONTAINER_TOOL)"	
+ifdef DOCKER_CMD
+	mkdir -p data
```

Comment:
This `mkdir` should be on the podman side of the if else condition since `docker run` creates the directory if it doesn't exist.

Actually we can remove the if else block since `mkdir -p data` will succeed even if the `data` directory already exists.